src/classes/RD_RecurringDonations_TDTM.cls

```diff
@@ -86,6 +86,11 @@ public class RD_RecurringDonations_TDTM extends TDTM_Runnable {
                      r.npe03__Open_Ended_Status__c != system.label.npe03.RecurringDonationClosedStatus)) {
                     r.addError(system.label.npe03.RecurringDonationTooManyInstallmentsError);
                 }            
+                // Reset this field to false if the installment period is not Monthly or Quarterly
```

Comment:
is there a case to be made that we should support last day of month for custom intervals.  For example, what if they define one for Every 2 months or every 6 months?